apps/mull-ui/src/app/pages/create-event/location-autocomplete/location-autocomplete-textbox.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState, ChangeEvent, useEffect, useMemo } from 'react';
```

Comment:
One problem I've been having is that the modal closes whenever I delete all of the content of the text field. This gets really annoying when I've already confirmed an address and then I want to change it. I can't delete the content in the text field and write a new address before the modal closes on me.

Reply:
Yeah i think the intention was that you would always be forced to put in a new valid address but a problem occurs lets say if u wanted to change the "750 street" to "850 street" , the moment u delete "7" it will close the modal making it impossible to ever write 8 in the modal unless u refresh the page. 

This was something i had brought up, and a potential solution was to have a clear (x) button on the autocomplete-textbox (mockups would need to be updated for this small change). Let me know what you guys think.

Reply:
No I don't agree, the solution would be to not close the modal when the last character is deleted. I don't know how that would be done since I didn't pay too much attention on the implementation of the text box but its 100% possible

Reply:
Not saying that having a clear button is a bad idea, but its not needed, and its just a band-aid for a bug that should be easy to fix

Reply:
At this point can we just remove the modal? It complicates things 2000% xD

Reply:
My solution is to have a simple searchable select, and that will remove material UI altogether

Reply:
Ok sure, i think we'll go that approach then @TheGreatMarkus, should be a simple fix

Reply:
I don't think the modal is an issue. This is most likely related to the logic used to close the modal.

Reply:
yeah but there's also the fact that the location cannot be sent (probably because of formik), and Idk if it's worth the extra time to patch all of this instead of creating a simple searchable select

Reply:
This was fixed for me just fyi